src/marqo/core/unstructured_vespa_index/unstructured_vespa_index.py

```diff
@@ -210,59 +216,46 @@ def tree_to_filter_string(node: search_filter.Node) -> str:
         if marqo_query.filter is not None:
             return tree_to_filter_string(marqo_query.filter.root)
 
-    @staticmethod
-    def _get_score_modifiers(marqo_query: MarqoQuery) -> \
-            Optional[Dict[str, Dict[str, float]]]:
-        if marqo_query.score_modifiers:
-            mult_tensor = {}
-            add_tensor = {}
-            for modifier in marqo_query.score_modifiers:
-                if modifier.type == ScoreModifierType.Multiply:
-                    mult_tensor[modifier.field] = modifier.weight
-                elif modifier.type == ScoreModifierType.Add:
-                    add_tensor[modifier.field] = modifier.weight
-                else:
-                    raise InternalError(f'Unknown score modifier type {modifier.type}')
-
-            # Note one of these could be empty, but not both
-            return {
-                unstructured_common.QUERY_INPUT_SCORE_MODIFIERS_MULT_WEIGHTS: mult_tensor,
-                unstructured_common.QUERY_INPUT_SCORE_MODIFIERS_ADD_WEIGHTS: add_tensor
-            }
+    def _get_lexical_search_term(self, marqo_query: MarqoLexicalQuery) -> str:
```

Comment:
Is this method changed? Is this just the change of order and some comments?

Reply:
I just extracted it, since it was under `_to_vespa_lexical_query` before, but now we need it for hybrid. Also added a check to determine whether to use `marqo_query.hybrid_parameters.scoreModifiersLexical` or `marqo_query.score_modifiers`